app/src/main/java/com/example/euriskocodechallenge/data/model/User.kt

```diff
@@ -1,16 +1,15 @@
 package com.example.euriskocodechallenge.data.model
 
-import android.graphics.Bitmap
 import androidx.room.Entity
 import androidx.room.PrimaryKey
 
 @Entity(tableName = "user")
 data class User(
     @PrimaryKey(autoGenerate = true)
-    val userId: Long = 0L,
-    val email: String,
-    val fName: String,
-    val lName: String,
-    val password: String,
-    val imageSrc: Bitmap
+    var userId: Long = 0L,
+    var email: String,
+    var fName: String,
```

Comment:
@martin371998 you should name your variables relevant to their content (e.g. firstName, lastName)